app/Resources/views/macros/layout.html.twig

```diff
@@ -45,3 +45,37 @@
         -#}<span class="glyphicon glyphicon-trash"></span>{#-
     -#}</a>
 {% endmacro %}
+
+{% macro statsBlock(event, title, metrics, extraTitle) %}
+    <article class="event-stats-block event-stats--{{ title }} clearfix">
+        <header class="col-sm-12">
+            <h3>
+                {{ msg(title) }}
+                {% if extraTitle is defined %}
+                    {{ extraTitle }}
+                {% endif %}
+            </h3>
+        </header>
+
+        {% for metric in metrics %}
+            {% set stat = event.statistic(metric) %}
+            {% if stat is not null %}
+                <dl class="col-sm-2">
+                    <dt>
+                        {{ msg(metric, [stat.offset]) }}
+                        {% if msg_exists(stat.metric ~ '-desc', [stat.offset]) %}
+                            <span class="font-awesome info-icon"
+                                  data-toggle="tooltip"
+                                  data-placement="top"
+                                  title="{{ msg(stat.metric ~ '-desc', [stat.offset]) }}"
+                            ></span>
```

Comment:
What's this character meant to be? It's not valid.

Reply:
How did that get in there!! Fixed